### PR TITLE
Create catalogue item disabled

### PR DESCRIPTION
**Pull request by cscwooller**

### CODE REVIEW

File: src/clj/rems/db/test_data.clj [hunk 1]
```diff
@@ -445,7 +445,7 @@
 
 (defn- create-catalogue-item! [resource workflow form localizations]
   (let [id (:id (db/create-catalogue-item!
-                 {:title "non-localized title" :resid resource :wfid workflow :form form}))]
+                 {:title "non-localized title" :resid resource :wfid workflow :form form :state "enabled"}))]
```
Comment (Macroz) [inline src/clj/rems/db/test_data.clj]: Isn't `enabled` now the default?

File: resources/sql/queries.sql [hunk 2]
```diff
@@ -45,8 +45,14 @@ WHERE ci.id = :item
...
 -- :doc Create a single catalogue item
 INSERT INTO catalogue_item
-(title, formid, resid, wfid)
-VALUES (:title, :form, :resid, :wfid)
+(title, formid, resid, wfid, state)
+VALUES (:title, :form, :resid, :wfid,
+/*~ (if (:state params) */
+CAST(:state as item_state)
+/*~*/
+CAST('enabled' as item_state)
```
Comment (Macroz) [inline resources/sql/queries.sql]: Shouldn't need a `CAST` here?

File: src/clj/rems/db/test_data.clj (hunk 1 above)
Comment (cscwooller) [reply src/clj/rems/db/test_data.clj]: Yes, but it is on lower levet, in queries.sql

File: resources/sql/queries.sql (hunk 2 above)
Comment (cscwooller) [reply resources/sql/queries.sql]: True.

File: src/clj/rems/db/test_data.clj (hunk 1 above)
Comment (Macroz) [reply src/clj/rems/db/test_data.clj]: But doesn't `db/create-catalogue-item!` here refer to the db fn?

**Pull request merged**